src/editor/core/plugins/source/menu.js

```diff
@@ -17,10 +17,10 @@ import {
 const switchMode = function (context) {
     return new MenuItem({
         id: 'source',
-        title: "Swtich editor mode",
+        title: "Switch editor mode",
```

Comment:
Such messages must be translatable. Is this already included?

Reply:
Can you please check if more translatable strings have been added?
It would be also good if you create a related PR in the core project. 

Please do not change the message files manually. We have a translation community here translate.humhub.org - direct modification will lead to conflicts.